### PR TITLE
vala: update to 0.40.8

### PR DESCRIPTION
**Pull request by Adsun701**
This updates vala to 0.40.8.

### CODE REVIEW

**Comment by lazka** [comment]
@Adsun701 have you actually tested if it works? Last time I tried updating it failed because it installs some new library not in PATH.

For example, does "valac --help" work?

**Comment by Adsun701** [comment]
@lazka OK, it does not work. But I noticed that libvalaccodegen.dll is installed to ${MINGW_PREFIX}/lib/bin and not ${MINGW_PREFIX}/bin, so I will check that out right now.

**Comment by Adsun701** [comment]
Confirmed it was libvalaccodegen.dll being installed to the wrong place. Now fixed.

**Comment by lazka** [comment]
thanks, looks good!

File: mingw-w64-vala/PKGBUILD [hunk 1]
```diff
@@ -42,6 +46,9 @@ build() {
 package() {
   cd "${srcdir}/build-${MINGW_CHOST}"
   make -j1 DESTDIR="${pkgdir}" install
-
+  
+  # Move libvalaccodegen.dll to correct directory
+  mv "${pkgdir}${MINGW_PREFIX}"/lib/bin/libvalaccodegen.dll "${pkgdir}${MINGW_PREFIX}"/bin/libvalaccodegen.dll
```
Comment (Alexpux) [inline mingw-w64-vala/PKGBUILD]: More general will be move *.dll and remove lib/bin folder

**Pull request merged**